src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -189,6 +190,21 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
           return !!artwork.increased_interest_signal
         },
       },
+      runningShowOrFair: {
```

Comment:
This field seems to pretty clearly return a show, so I'm wondering what the `orFair` naming refers to? Is it that the show might be a fair booth? If so - maybe `runningShowOrFairBooth` - but also probably just `runningShow` might be sufficient.